bin/build-package-containing-file.js

```diff
@@ -0,0 +1,39 @@
+const fs = require( 'fs' );
+const { spawnSync } = require( 'child_process' );
+
+const args = process.argv.slice( 2 );
+if ( args.length < 1 ) {
+	console.error( 'failed to rebuild package: at least one filename must be provided' );
+	process.exit( 1 );
+}
+const fileName = args[ 0 ];
+if ( ! fileName ) {
+	console.error( 'failed to rebuild package: missing filename' );
+	process.exit( 1 );
+}
+const packageDirectoryMatches = fileName.match( /^packages\/[^/]+/ );
+if ( packageDirectoryMatches.length < 1 ) {
+	console.error( 'failed to rebuild package: missing package directory' );
+	process.exit( 1 );
+}
+const packageDirectory = packageDirectoryMatches[ 0 ];
+if ( ! packageDirectory ) {
+	console.error( 'failed to rebuild package: missing package directory contents' );
+	process.exit( 1 );
+}
+const packageJSONFile = packageDirectory + '/package.json';
+const rawData = fs.readFileSync( packageJSONFile );
+const parsedData = JSON.parse( rawData );
```

Comment:
You can just `const parsedData = require(packageJSONFile)`
```suggestion
const parsedData = require(packageJSONFile)
```

Reply:
When I do this, I get a fatal error:

```
Error: Cannot find module 'packages/composite-checkout/package.json'
```

Reply:
Ah right, it needs a `./` at the front. Interesting that `readFileSync` is ok with it, TIL.